src/index.html

```diff
@@ -13,5 +13,159 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+          Personal Information
+        </legend>
+        <div class="form-field">
+          Surname:&nbsp;
+          <input id="surname"
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
```

Comment:
If multi-line, 1st attribute also starts from the new line
```suggestion
          <input
            id="surname"
            type="text"
            name="Surname"
            autocomplete="off"
          >
```